.github/workflows/pytest.yml

```diff
@@ -0,0 +1,34 @@
+name: pytest
+
+# This workflow will
+# run all tests with pytest  
+on: [push]
+
+jobs:
+  
+  tests:
+   runs-on: ubuntu-18.04
+   strategy:
+      matrix:
+        os: [ubuntu-18.04]
+        python-version: [3.6, 3.7, 3.8, 3.9]
+   steps:
+     
+    - name: Checkout
+      uses: actions/checkout@v2
+      with:
+        fetch-depth: 0 # otherwise, you will failed to push refs to dest repo
+
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}    
+
+    - name: Test python version [3.6, 3.7, 3.8, 3.9] and install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install pytest pytest-cov
+
+    - name: Run Tests and create coverage
+      run: |
+        pytest -v --cov=reportdaily tests/
```

Comment:
I'd recommend to move all options into the `setup.cfg` file and call just `pytest` or `pytest -v`.  That way you can adapt the config file without changing this file.